plugins/alerting/pkg/alerting/api_endpoints.go

```diff
@@ -0,0 +1,250 @@
+package alerting
```

Comment:
Check for extra blank lines in this file

Reply:
Done